.gitignore

```diff
@@ -104,6 +104,7 @@ deps/npm/node_modules/.bin/
 /SHASUMS*.txt*
 
 # test artifacts
+package-lock.json
```

Comment:
There is also a `package-lock.json` in tools/doc..

Reply:
ack.
renamed.